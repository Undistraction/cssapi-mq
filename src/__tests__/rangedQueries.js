import { mqWithValidBreakpointsForRange } from './testHelpers/data'

describe(`rangedQueries`, () => {
  it(`generates ranged queries`, () => {
    const mq = mqWithValidBreakpointsForRange(`width`)
    const result = mq.queryAboveWidth(`medium`)``
    expect(result).toMatchSnapshot()
  })
})
