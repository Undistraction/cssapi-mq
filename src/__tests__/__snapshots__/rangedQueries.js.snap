// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rangedQueries generates ranged queries 1`] = `
Array [
  "
  ",
  "@media (min-width: 56.25em)",
  " {
    ",
  ";
  }
",
]
`;
